client/app/dedicated/server/interfaces/ola-step-checker/ola-step-checker.controller.js

```diff
@@ -0,0 +1,46 @@
+import _ from 'lodash';
+
+export default class {
+  /* @ngInject */
+  constructor($location, $state) {
+    this.$location = $location;
+    this.$state = $state;
```

Comment:
Should use `resolve`.

Reply:
In a component that not use any routing ? Or in the resolve of interface.routing.js ?

Reply:
Should be injected. If you need inject-it to parent component to inject-it to this component.